x-pack/plugins/cross_cluster_replication/public/app/app.js

```diff
@@ -45,10 +45,10 @@ export class App extends Component {
     return (
       <div>
         <Switch>
-          <Redirect exact from={`${BASE_PATH}`} to={`${BASE_PATH}/auto_follow_patterns`} />
+          <Redirect exact from={`${BASE_PATH}/:section`} to={`${BASE_PATH}`} />
           <Route exact path={`${BASE_PATH}/auto_follow_patterns/add`} component={AutoFollowPatternAdd} />
           <Route exact path={`${BASE_PATH}/auto_follow_patterns/edit/:id`} component={AutoFollowPatternEdit} />
-          <Route path={`${BASE_PATH}/:section`} component={CrossClusterReplicationHome} />
+          <Route path={`${BASE_PATH}`} component={CrossClusterReplicationHome} />
```

Comment:
@sebelga How do you feel about this routing change? I didn't think the `/auto_follow_patterns` route made sense since the page title is "Cross Cluster Replication" and it's the CCR app landing page.

Reply:
@cjcenizal Yes I think this could work

Reply:
Based on conversation I will revert this change, with the intention of having two paths (one per tab) in the future: `/auto_follow_patterns` and `/follower_indices`.